src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+  const filteredItems = sourceString.split(';').filter(x => x.includes(':'));
+
+  filteredItems.forEach((element) => {
+    const part = element.split(':');
+    const trimKey = part[0].trim();
+    const trimValue = part[1].trim();
```

Comment:
you can destructure those from part
```suggestion
    const [key, value] = part
```